lib/ts/recipe/session/recipe.tsx

```diff
@@ -145,9 +166,20 @@ export default class Session extends RecipeModule<unknown, unknown, unknown, Nor
                 userContext,
             });
 
+            if (failureRedirectInfo.redirectPath === null) {
```

Comment:
is this required? Cause failureRedirection never returns `null` right?

Reply:
Good catch! I have removed it now.